assets/scss/layouts/products/_productSaleBadges.scss

```diff
@@ -22,11 +26,20 @@
     z-index: zIndex("lower");
 }
 
-.sale-flag-star:before {
+.sold-out-flag-star,
+.sold-out-flag-star:before,
+.sold-out-flag-star:after {
+    @extend .sale-flag-star;
```

Comment:
Could we make a class base class for start and reuse it for sale and sold out classes?

Reply:
I added placeholder selector that is used as base class for sales and sold-out classes